staff/YaninaGomezCucchiararo/css-layout-simple-adaptive/style/main-desktop-css.css

```diff
@@ -0,0 +1,54 @@
+html, body {
+    min-height:100% !important;
+    width: 100%;
+    height: 100%;
+}
+
+*{
+    box-sizing: border-box;
+    padding: 0%;
+    margin: 0%;
+    color:white;
+    text-align: center;
+    border: 1px solid white;
+}
+
+header{
+    height: 20%;
+    width: 100%;
+    background-color:black
+}
+
+main{
+    height: 60%;
+    width:100%;
+    background-color: black;
```

Comment:
TIP same css rules for different selectors may be grouped

SEE Grouping Selectors https://www.w3schools.com/css/css_syntax.asp